KsApi/models/templates/CommentRepliesEnvelopeTemplates.swift

```diff
@@ -17,11 +17,11 @@ extension CommentRepliesEnvelope {
     totalCount: 14
   )
 
-  static let singleReplyTemplate = CommentRepliesEnvelope(
+  static let failedAndSuccessRepliesTemplate = CommentRepliesEnvelope(
     comment: .template,
     cursor: "cursor",
     hasPreviousPage: false,
-    replies: [.replyTemplate],
-    totalCount: 1
+    replies: [.replyFailedTemplate, .replyTemplate],
```

Comment:
Not sure if I'm overthinking this but looking at the screenshots I'm wondering if the order of these two should be swapped? Is there a situation where a failed reply is above a successful one in a given thread?

Reply:
Ah well it's certainly possible if a user simply does not retry the failed one and then manages to send a successful one.

Reply:
Yea, it's unlikely, because it's easier to a failed comment before retyping a new one. There could be a situation where optimistically the comment is posted, the user posts another because they thing the first one was successful, then the first comment shows failed.

Reply:
Good call making it a common style, it fixes that `top` padding of `1` when the cell was not indented (above).